src/views/ont/ONTSampleIndex.vue

```diff
@@ -73,6 +73,7 @@ export default {
         { key: 'cost_code', label: 'Cost code' },
         { key: 'external_study_id', label: 'External study ID' },
         { key: 'location', label: 'Location', sortable: true },
+        { key: 'sample_retention_instruction', label: 'Retention Instruction', sortable: true },
```

Comment:
You could pass in a formatter key in this object, e.g. `{ key: 'sample_retention_instruction', ..., formatter: (instruction) => formatRetentionInstruction(instruction) }` instead of formatting on the displayedRequests. Two ways of doing the same thing so I think either is fine.

existing example: https://github.com/sanger/traction-ui/blob/develop/src/views/pacbio/PacbioLibraryIndex.vue#L162

Reply:
Yes, good point.
 The locationBuilder retrieves and formats the 'Location' column a bit differently by using requests and locationsData which applies on the entire array, which is why I thought it might be useful to combine both 'location' and 'retention_instruction' formatting into a shared utility thereby it can be reused across both PacBio and ONT. Hope that makes sense.